src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,149 @@
+import {
+  cloneElement,
+  KeyboardEvent,
+  ReactElement,
+  ReactNode,
+  useState
+} from 'react';
+
+import castArray from 'lodash/castArray';
+
+import Box from '../Box';
+import Flex from '../Flex';
+import StyledTabs, { Props as StyledTabProps } from './StyledTabs';
+import { Props as TabProps } from './Tab';
+import { ENTER_KEY, SPACEBAR } from '../constants';
+
+type Tab = ReactElement<TabProps>;
+
+const getActiveTab = (
+  activeTabId: string | undefined,
+  children: Array<Tab | null>
```

Comment:
I don't know why we're supporting `null` children.  This can probably be removed as long as we update tests and storybook stories.